### ISSUE TITLE
modify dangerous-view-replace-wo-priority: Replace in QWeb template is not detected

### ISSUE THREAD

**Issue by luisg123v** [OCA/pylint-odoo#323]
The lint is not detecting replaces when they're inside QWeb templates 
`<template>`
instead of
`<record ... model="ir.ui.view">`
for instance:

```
<template id="some_qweb_template" inherit_id="web.some_qweb_template">
    <xpath expr="//div[@name='some_div']" position="replace">
        ...
    </xpath>
</template>
```

**Comment by luisg123v** [comment]
CC @moylop260 @carmenmiranda
